docs/secondary-network.md

```diff
@@ -170,7 +206,103 @@ spec:
    image: antrea/toolbox:latest
 ```
 
-**At the moment, we do NOT support annotation update / removal: when the
+### SR-IOV
+
+#### Creating network functions
+
+To configure SR-IOV network for Pods, the required network functions must be
```

Comment:
Is it called "network function"? Should we use "SR-IOV Virtual Functions"?

Reply:
It's called network functions in this doc: https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin?tab=readme-ov-file#creating-network-functions because there could be two types of network functions: SR-IOV Virtual Functions and Subfunctions (specific to Auxiliary device), but we never tested the latter and don't have the resource to test it.

Do you think we should make out doc specific to SR-IOV VF?

Reply:
Humm.. in this case, we should make it clear that SF should work but is not verified?

Reply:
done